docs/content/distributing-bundles.md

```diff
@@ -3,11 +3,75 @@ title: Distributing Bundles
+
+Once you have provided values for these, run the `porter build` command one last time to verify that your invocation image can be successfully built and to ensure that the value you specified in `invocationImage` is correct.
+
+Next, run the `porter publish` command in order to push the invocation image to the specified repository and to regenerate a CNAB bundle manifest using this newly pushed image. You should see output like the following:
+
+```
+$ porter publish
+Pushing CNAB invocation image...
+The push refers to repository [docker.io/deislabs/porter-kubernetes]
+c412023fe7ea: Preparing
+397a70d3e67f: Preparing
+49037d9d1b30: Preparing
+c7956a703d1e: Preparing
+c581f4ede92d: Preparing
+c581f4ede92d: Layer already exists
+c7956a703d1e: Layer already exists
+49037d9d1b30: Layer already exists
+c412023fe7ea: Layer already exists
+397a70d3e67f: Layer already exists
+latest: digest: sha256:d8aa654f5e60d64f698d79664480500b8de469a22e15dc69806e8172848e17d6 size: 1370
+
+Generating CNAB bundle.json...
+
+Generating Bundle File with Invocation Image deislabs/porter-kubernetes@sha256:d8aa654f5e60d64f698d79664480500b8de469a22e15dc69806e8172848e17d6 =======>
+Generating parameter definition porter-debug ====>
+Generating credential kubeconfig ====>
+Starting to copy image deislabs/porter-kubernetes@sha256:d8aa654f5e60d64f698d79664480500b8de469a22e15dc69806e8172848e17d6...
+Completed image deislabs/porter-kubernetes@sha256:d8aa654f5e60d64f698d79664480500b8de469a22e15dc69806e8172848e17d6 copy
+WARN[0005] reference for unknown type: application/vnd.cnab.config.v1+json
+Pushed successfully, with digest "sha256:57c34a53e84607562e396280563186759139454d1704c727180aac1819b75a4f"
+```
+
+Note: you can safely ignore the `WARN[0005] reference for unknown type: application/vnd.cnab.config.v1+json` message.
+
+When this command is complete, your CNAB bundle manifest and invocation image will have been successfully pushed to the specified OCI registry. It can then be installed with the `porter install` command:
+
+```
+$ porter install --tag deislabs/porter-kube-bundle:1.0 -c kool-kred
```

Comment:
It would help for the documentation to use example names that have some meaning or hint at what people should use instead. I suggest using something either obviously fake like "foo" or better yet something like "--cred mycluster" or something like that for examples.

Also, I recommend using the full tag names, `--cred` and not the short flags in the examples, for consistency.